config/default.yml

```diff
@@ -860,7 +860,10 @@ Layout/FirstMethodArgumentLineBreak:
                  multi-line method call.
   Enabled: false
   VersionAdded: '0.49'
+  VersionChanged: '<<next>>'
```

Comment:
Since no configuration changes have been made that would alter existing behavior, `VersionChanged '<<next>>'` is likely unnecessary.

Reply:
Sure, removed.